src/components/microlearning/VideoDetailPage.jsx

```diff
@@ -87,19 +87,23 @@ const VideoDetailPage = () => {
       <Row>
         <article className="col-12 col-lg-9">
           <div className="d-flex flex-column align-items-start flex-grow-1 video-container">
-            <div className="d-flex flex-row align-items-center justify-content-between">
-              <h2 data-testid="video-title" className="mb-0">
-                {videoData?.courseTitle}
-              </h2>
+            <div className="d-flex flex-row align-items-start justify-content-between">
```

Comment:
Maybe we should hide this entire section if `courseTitle` is empty?

Reply:
That's very unlikely because if `courseTitle` is empty, we use `videoTitle` for that field. However, I can add a check.